examples/others/fragment/lv_example_fragment_1.c

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
`lv_example_fragment_1` results in empty screen for me.

Reply:
Done, got it fixed.